src/content/docs/contributing/modules.md

```diff
@@ -842,6 +842,23 @@ The key words "MUST", "MUST NOT", "SHOULD", etc. are to be interpreted as descri
     """
     ```
 
+11. Occasionally, some tools do not exit with the expected exit code 0 upon successful use of the tool. In these cases one can use
+    the `||` operator to run another useful command when the exit code is not 0 (for example, testing if a file is not size 0).
+
+    ```nextflow
```

Comment:
```suggestion
    ```nextflow {4}
```

Reply:
What does this do?

Reply:
highlight a specific line in the codeblock: https://nf-co.re/docs/contributing/website_markdown#highlight-lines